app/objects/secondclass/c_link.py

```diff
@@ -66,6 +66,8 @@ def __init__(self, operation, command, paw, ability, status=-3, score=0, jitter=
         self.collect = None
         self.finish = None
         self.facts = []
+        if ability.payload:
```

Comment:
this is not the appropriate place to put this. not sure where offhand.. probably where we add the link to the operation chain instead..